tests/mobile/login /login.specs.js

```diff
@@ -0,0 +1,29 @@
+const auth = require('../../auth')();
```

Comment:
This path has a space at the end of `/login /` - can you move it to remove the space?

Reply:
Fixed